src/handwritten/__main__.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+import click
+
+from . import _install_daqmx
+
+from typing import Optional
+
+
+
+@click.group("nidaqmx")
```

Comment:
What is this group name used for? I don't think I've seen anyone set a group name for a top-level group before.

If I change the group name, `click` still prints "nidaqmx.cmd":
```
PS D:\dev\nidaqmx-python> poetry run nidaqmx installdriver --help
Usage: nidaqmx.cmd installdriver [OPTIONS]

Options:
  --help  Show this message and exit.
```

I guess that's because the file in `.venv\Scripts` is a `.cmd` file.

`python -m nidaqmx` doesn't seem to use the group name either.

Reply:
ok